test/parallel/test-timers-refresh.js

```diff
@@ -44,6 +44,87 @@ const { setUnrefTimeout } = require('internal/timers');
   strictEqual(timer.refresh(), timer);
 }
 
+// passing 3 arguments to setUnrefTimeout
```

Comment:
This could be instead made into a loop for 3, 4, 5 & 6 arguments so that there's not so much code duplication.

Reply:
Thanks! using loop is better. I'll try to fix it.